### PR TITLE
README.md per issue 11

### LINKED ISSUE #11: A README file needed
We need a `README` file for new users. A modified version of current [Android app one](https://github.com/mlcommons/mobile_app_open/blob/master/android/README.md) should be enough.

### PR DESCRIPTION
**Pull request by relja128**
Closes #11

### CODE REVIEW

**Comment by relja128** [comment]
I used the top half (overview) of the old readme since the full readme is in the Android folder.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+While the initial version of the app uses TensorFlow Lite as the default
+inference engine, the plan is to support addition of alternative inference
+frameworks contributed by the broader MLPerf community.
+
+## Folder Structure
+
+* android - Contains the Android version of the app
+* mobile_back_qti - QTI backend for Android
+* mobile_back_samsung - Samsung backend for Android
+* mobile_back_tflite - Combined TFLite / MediaTek backends for Android
```
Comment (mcharleb) [inline README.md:46]: We will need to add Pixel once Pixel backend is merged

Comment (mcharleb) [reply README.md:46]: @relja128  I suggest we merge pixel backend first then update this PR to add Pixel and then merge this PR.

**Review by mcharleb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stegben) [inline README.md:46]:
```suggestion
* mobile_back_pixel - Pixel backend using NNAPI for Android
* mobile_back_tflite - Combined TFLite / MediaTek backends for Android
```

**Review by mcharleb — APPROVED** [review]

**Pull request merged**